File: library/src/main/java/com/example/library/dto/response/LibrarianResponse.java
package com.example.library.dto.response;

import java.time.LocalDateTime;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class LibrarianResponse {

    private Integer id;

    private String username;

    private String fullName;

    private String role;

    private String staffCode;
    
    private LocalDateTime updatedAt;
    
    private LocalDateTime deleteAt;
}

File: MyApplication/app/src/main/java/com/example/myapplication/data/remote/dto/response/LibrarianResponse.kt
package com.example.myapplication.data.remote.dto.response

import java.time.LocalDateTime

data class LibrarianResponse(
    val id: Int,
    val username: String,
    val fullName : String,
    val role: String,
    val staffCode : String,
    val updatedAt: LocalDateTime,
    val deleteAt: LocalDateTime?
)